### ISSUE TITLE
types error: quasar 1.7.0 types/index.d.ts => LoadingBar.setDefaults

### ISSUE THREAD

**Issue by a599155162** [quasarframework/quasar#6088]
**Type definition error**
quasar\dist\types\index.d.ts(308,24)  TS1005: ',' expected.

```
export interface LoadingBar {
    /**
     * Notify bar you've started a background activity
     * @param speed Delay (in milliseconds) between bar progress increments
     */
    start (speed? : number): void
    /**
     * Notify bar one background activity has finalized
     */
    stop (): void
    /**
     * Manually trigger a bar progress increment
     * @param amount Amount (0.0 < x < 1.0) to increment with
     */
    increment (amount? : number): void
    /**
     * Set the inner QAjaxBar's props
     * @param ...props QAjaxBar component props
     */
    setDefaults ([index: string]: any): void  // [index, string] ???
}
```

**Comment by a599155162** [comment]
env: typescript 3.7.2+

**Comment by rstoenescu** [comment]
Thanks for reporting!
Fix will be available shortly in "quasar" v1.7.1.
